Cargo.toml

```diff
@@ -8,474 +8,474 @@ license = "GPL-3.0-only"
-	"substrate/primitives/consensus/common",
-	"substrate/primitives/consensus/grandpa",
-	"substrate/primitives/consensus/pow",
-	"substrate/primitives/consensus/sassafras",
-	"substrate/primitives/consensus/slots",
-	"substrate/primitives/core",
-	"substrate/primitives/core/fuzz",
-	"substrate/primitives/core/hashing",
-	"substrate/primitives/core/hashing/proc-macro",
-	"substrate/primitives/crypto/ec-utils",
-	"substrate/primitives/database",
-	"substrate/primitives/debug-derive",
-	"substrate/primitives/externalities",
-	"substrate/primitives/genesis-builder",
-	"substrate/primitives/inherents",
-	"substrate/primitives/io",
-	"substrate/primitives/keyring",
-	"substrate/primitives/keystore",
-	"substrate/primitives/maybe-compressed-blob",
-	"substrate/primitives/merkle-mountain-range",
-	"substrate/primitives/metadata-ir",
-	"substrate/primitives/mixnet",
-	"substrate/primitives/npos-elections",
-	"substrate/primitives/npos-elections/fuzzer",
-	"substrate/primitives/offchain",
-	"substrate/primitives/panic-handler",
-	"substrate/primitives/rpc",
-	"substrate/primitives/runtime-interface",
-	"substrate/primitives/runtime-interface/proc-macro",
-	"substrate/primitives/runtime-interface/test-wasm-deprecated",
-	"substrate/primitives/runtime-interface/test-wasm",
-	"substrate/primitives/runtime-interface/test",
-	"substrate/primitives/runtime",
-	"substrate/primitives/session",
-	"substrate/primitives/staking",
-	"substrate/primitives/state-machine",
-	"substrate/primitives/statement-store",
-	"substrate/primitives/std",
-	"substrate/primitives/storage",
-	"substrate/primitives/test-primitives",
-	"substrate/primitives/timestamp",
-	"substrate/primitives/tracing",
-	"substrate/primitives/transaction-pool",
-	"substrate/primitives/transaction-storage-proof",
-	"substrate/primitives/trie",
-	"substrate/primitives/version",
-	"substrate/primitives/version/proc-macro",
-	"substrate/primitives/wasm-interface",
-	"substrate/primitives/weights",
-	"substrate/scripts/ci/node-template-release",
-	"substrate/test-utils",
-	"substrate/test-utils/cli",
-	"substrate/test-utils/client",
-	"substrate/test-utils/runtime",
-	"substrate/test-utils/runtime/client",
-	"substrate/test-utils/runtime/transaction-pool",
-	"substrate/utils/binary-merkle-tree",
-	"substrate/utils/build-script-utils",
-	"substrate/utils/fork-tree",
-	"substrate/utils/frame/benchmarking-cli",
-	"substrate/utils/frame/frame-utilities-cli",
-	"substrate/utils/frame/generate-bags",
-	"substrate/utils/frame/generate-bags/node-runtime",
-	"substrate/utils/frame/remote-externalities",
-	"substrate/utils/frame/rpc/client",
-	"substrate/utils/frame/rpc/state-trie-migration-rpc",
-	"substrate/utils/frame/rpc/support",
-	"substrate/utils/frame/rpc/system",
-	"substrate/utils/frame/try-runtime/cli",
-	"substrate/utils/prometheus",
-	"substrate/utils/wasm-builder",
- 	"cumulus/parachains/integration-tests/emulated/networks/rococo-westend-system",
+  "bridges/bin/runtime-common",
```

Comment:
Is it possible to use a tab here? Not sure if its strictly better, but the diff will be smaller.

Reply:
We use spaces elsewhere else, personally I think fine to just bite the bullet now and then use them moving forward

Reply:
Where were we using tabs? Other cargo.toml files I saw used spaces, and we indent our code with spaces.

Anyway not a big deal at all :P I updated the tapio config to change indentation to tabs.

Reply:
We indent our code with tabs. I would check again ;) 

Also looking at the diff, you are seeing that we used more tabs in toml files than spaces :P